src/validate.rs

```diff
@@ -392,12 +392,10 @@ pub fn validate_iterator_fields(
     tooltipped: &mut Tooltipped,
 ) {
     // undocumented
-    if list_type != ListType::None {
-        if vd.field_item("custom", Item::Localization) {
-            *tooltipped = Tooltipped::No;
-        }
-    } else {
+    if list_type == ListType::None {
```

Comment:
This makes it the wrong way around. Every if-else in this section is "if we should validate some fields, validate them, else ban them". I can't help that clippy wants to change it the wrong way around :)